### DISCUSSION TITLE
Curl version is out of date(7.74.0 OpenSSL/1.1.1n)

### DISCUSSION THREAD

**Discussion by topias11a705** [docker-library/wordpress#821] [General]
The version of Curl included in the container is out of date(7.74.0 OpenSSL/1.1.1n) and should be changed to include the latest CURL version.Can I include updates in the Dockerfile? No problem if you upgrade in any other way.

**Comment by topias11a705** [comment]
![image](https://user-images.githubusercontent.com/38270203/236314361-34c135b0-ea16-4aff-a109-40bb748f832a.png)

**Reply by topias11a705** [reply]
root@ubuntu:/home/user# docker-compose exec wordpress bash
root@3facad8ecc71:/var/www/html# curl -V

**curl 7.74.0** (aarch64-unknown-linux-gnu) libcurl/7.74.0 OpenSSL/1.1.1n zlib/1.2.11 brotli/1.0.9 libidn2/2.3.0 libpsl/0.21.0 (+libidn2/2.3.0) libssh2/1.9.0 nghttp2/1.43.0 librtmp/2.3
Release-Date: 2020-12-09
Protocols: dict file ftp ftps gopher http https imap imaps ldap ldaps mqtt pop3 pop3s rtmp rtsp scp sftp smb smbs smtp smtps telnet tftp
Features: alt-svc AsynchDNS brotli GSS-API HTTP2 HTTPS-proxy IDN IPv6 Kerberos Largefile libz NTLM NTLM_WB PSL SPNEGO SSL TLS-SRP UnixSockets

**Reply by topias11a705** [reply]
apt-get update && apt-get upgrade -y curl

**Reply by yosifkit** [reply]
The curl packages are as up-to-date as is available in Debian Bullseye.

```console
$ docker pull wordpress
...
Digest: sha256:06b3c3b2fdc126d5e28b1f1c78a99009fe186d7354c907074095d5661bd18570
Status: Downloaded newer image for wordpress:latest
docker.io/library/wordpress:latest
$ docker run -it --rm wordpress bash
root@ef876b1592f0:/var/www/html# dpkg -l | grep curl
ii  curl                          7.74.0-1.3+deb11u7             amd64        command line tool for transferring data with URL syntax
ii  libcurl4:amd64                7.74.0-1.3+deb11u7             amd64        easy-to-use client-side URL transfer library (OpenSSL flavour)
root@ef876b1592f0:/var/www/html# apt update
Get:1 http://deb.debian.org/debian bullseye InRelease [116 kB]
Get:2 http://deb.debian.org/debian-security bullseye-security InRelease [48.4 kB]
Get:3 http://deb.debian.org/debian bullseye-updates InRelease [44.1 kB]
Get:4 http://deb.debian.org/debian bullseye/main amd64 Packages [8183 kB]
Get:5 http://deb.debian.org/debian-security bullseye-security/main amd64 Packages [238 kB]
Get:6 http://deb.debian.org/debian bullseye-updates/main amd64 Packages [14.6 kB]
Fetched 8644 kB in 2s (5527 kB/s)
Reading package lists... Done
Building dependency tree... Done
Reading state information... Done
All packages are up to date.
root@ef876b1592f0:/var/www/html# apt list --upgradable
Listing... Done
root@ef876b1592f0:/var/www/html#
```

**Reply by topias11a705** [reply]
thankyou for your Reply
